__tests__/app.test.js

```diff
@@ -168,7 +168,68 @@ describe("GET /api/articles/:article_id/comments", () => {
   });
 });
 
+describe("POST", () => {
+  test("POST 201: adds a new comment using username and body from request", () => {
+    const testInput = {
+      username: "butter_bridge",
+      body: "really important words about thoughts"
+    };
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(testInput)
+      .expect(201)
+      .then(({ body }) => {
+        const { comment } = body;
+        expect(comment).toMatchObject({
+          comment_id: expect.any(Number),
+          body: expect.any(String),
+          author: expect.any(String),
+          article_id: expect.any(Number),
+          votes: expect.any(Number),
+          created_at: expect.any(String)
```

Comment:
This is a well written test, you can make the test even more robust by asserting for the specific values where you can, for example - 
Are we expecting `article_id` to be any old number or can we expect it to be `1` as that is the article we are posting to ?
Same goes for `body` and `author`, what would we expect those to be ?